### PR TITLE
a2ps test based on: https://progress.opensuse.org/issues/9472

### PR DESCRIPTION
**Pull request by dgutu-suse**
a2ps console test

### CODE REVIEW

File: tests/console/a2ps.pm [hunk 1]
```diff
@@ -0,0 +1,14 @@
+use strict;
+use base "consoletest";
+use testapi;
+use ttylogin;
+
+sub run() {
+    become_root;
+    assert_script_run("zypper -n in a2ps | tee /dev/$serialdev");
```
Comment (coolo) [inline tests/console/a2ps.pm]: if you use a pipe, the assert_script_run is useless - the exit status will be very likely correct as the tee is pretty easy to please

File: main.pm [hunk 2]
```diff
@@ -423,6 +423,7 @@ sub load_consoletests() {
             loadtest "console/zypper_ar.pm";
         }
         loadtest "console/zypper_ref.pm";
+        loadtest "console/a2ps.pm"; # a2ps is available by default only in openSUSE*
```
Comment (coolo) [inline main.pm]: please move that comment into the test itself, the main.pm is openSUSE specific anyway

File: tests/console/a2ps.pm [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    assert_script_run("curl https://www.suse.com > /tmp/suse.html");
```
Comment (coolo) [inline tests/console/a2ps.pm]: why not use -o ?

File: tests/console/a2ps.pm [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    assert_script_run("a2ps -o /tmp/suse.ps /tmp/suse.html | tee /dev/$serialdev");
+    assert_screen "a2ps_saved", 5;
+}
```
Comment (coolo) [inline tests/console/a2ps.pm]: remove the timeout here, the default will do

**Comment by coolo** [comment]
ln -s ../os-autoinst/tools tools; ./tools/tidy

**Comment by coolo** [comment]
you still need to tidy

**Comment by nilxam** [comment]
may I change main.pm and move this test on preinstalled image? otherwise https://openqa.opensuse.org/tests/100667

**Comment by sysrich** [comment]
Yes, go ahead

**Comment by coolo** [comment]
Yes, I was thinking the same - I merged the test and thought main.pm is easily changed.

That it breaks staging wasn't on my horizon, sorry

**Comment by nilxam** [comment]
ok, I'll do that tomorrow and sort out main.pm eg. move yast2 users test under yast2_ui_test. :)

**Pull request merged**